### PR TITLE
Validator setup

### PR DESCRIPTION
**Pull request by lilleyse**
Rounded out the project structure and add some utility functions borrowed from https://github.com/AnalyticalGraphicsInc/3d-tiles-tools/pull/4.

For the sake of time I'm just going to merge this now, but let me know if there are any issues I should address.

### CODE REVIEW

**Comment by pjcozzi** [comment]
> For the sake of time I'm just going to merge this now, but let me know if there are any issues I should address.

It's OK to run fast and loose, but instead of merging into `master`, make a `validator-master` branch; otherwise, keep `master` "ready to ship."

File: validator/README.md [hunk 1]
```diff
@@ -1 +1,67 @@
...
+
+### validate
+
+Validates the input tileset.
+
+```
+node ./bin/3d-tiles-validator.js ./specs/data/Tileset/
+```
+```
+node ./bin/3d-tiles-tools.js gzip -i ./specs/data/Tileset/
```
Comment (pjcozzi) [inline validator/README.md]: Is this example needed here?

File: validator/README.md [hunk 2]
```diff
@@ -1 +1,67 @@
...
+```
+
+|Flag|Description|Required|
+|----|-----------|--------|
+|`-i`, `--input`|Input directory of the tileset.| :white_check_mark: Yes|
```
Comment (pjcozzi) [inline validator/README.md]: Is this optional (big +1 from me)?  The example above does not have it.

**Comment by lilleyse** [comment]
> It's OK to run fast and loose, but instead of merging into master, make a validator-master branch; otherwise, keep master "ready to ship."

Alright, I'll do that from now on.

File: validator/README.md (hunk 1 above)
Comment (lilleyse) [inline validator/README.md]: That was a copy-paste mistake, fixed now.

File: validator/README.md (hunk 2 above)
Comment (lilleyse) [inline validator/README.md]: I've changed the wording for this so it is optional.

**Pull request merged**